tests/sharding/run.sh

```diff
@@ -90,12 +90,21 @@ function run() {
     # `FIXME: the following case is not supported automatically now, try to support it later`
     # so we try to do this `pause-task` and `resume-task` in the case now.
     sleep 3
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "pause-task test"\
+        "\"result\": true" 3
+    # pause twice
     run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
         "pause-task test"\
         "\"result\": true" 3
     # wait really paused
-    # FIXME: `if !st.stageCAS(pb.Stage_Running, pb.Stage_Paused)` in `subtask.go` is not enough to indicate the real stage.
-    sleep 2
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "query-status test" \
+        "Paused" 2
+    run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "resume-task test"\
+        "\"result\": true" 3
+    # resume twice
```

Comment:
Should we add a test to test `stop-task` twice?

Reply:
add in ecd45e6